.github/container/nsys_jax/nsys_jax/analyses/communication.py

```diff
@@ -95,6 +96,33 @@ def format_bandwidth(data, collective):
             )
         )
 
+    collective_types = set()
+    summary_data = defaultdict(dict)
+    for collective, df in steady_state.communication.groupby(
+        ["Collective"]
+    ):
+        collective_types.add(collective)
+        summary_data[collective] = df["DurHiddenMsToDurMs"].mean()
+
+    collective_width = max(len("Collective"), max(len(f"{collective}") for collective in collective_types))
+    ratio_width = len("Mean HiddenToTotalMs")
+
+    print()
```

Comment:
I think python `prettyTable` (https://pypi.org/project/prettytable/) should work here and you don't need to do all these calculations manually

Reply:
Done. Thanks!